packages/psss/src/components/EditableTable.js

```diff
@@ -42,6 +42,16 @@ const ReadOnlyTextField = styled(EditableTextField)`
   }
 `;
 
+const RedTooltip = styled(Tooltip)`
```

Comment:
Is `ErrorTooltip` a better name, in case the theme changes the error colour (highly unlikely, but possible)?